### PR TITLE
Our service

### PR DESCRIPTION
**Pull request by Honeylii**

### CODE REVIEW

File: domain/our-services/index.tsx
Comment (beedyboy) [inline domain/our-services/index.tsx:1]: One observation, you don't need to import these images

**Review by beedyboy — CHANGES_REQUESTED** [review]

File: components/formTemplate/index.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      <div className="w-full flex justify-between items-center">
+        <button
+          onClick={() => setView((prev) => (prev === 1 ? 2 : 1))}
+          className={`self-start w-[150px] md:px-6 py-2 p-2 ${
+            view === 2
+              ? "bg-[#fff] border-[1px] text-[#3BAD6B] border-[#3BAD6B]"
+              : " bg-[#3BAD6B] text-[#fff] "
+          }  shadow-md rounded-lg `}
+        >
+          {view === 1 ? "Next" : "Pervious"}
```
[the remark this answers is not in the document]
Comment (Honeylii) [reply components/formTemplate/index.tsx]: Fixed

File: domain/our-services/index.tsx
Comment (Honeylii) [reply domain/our-services/index.tsx:1]: Noted

File: domain/our-services/index.tsx [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+        {/*  */}
+
+        <ServiceLayout bgColor="white">
+          <div className="p-6 md:p-0 w-full flex flex-col items-center md:flex-row gap-3 md:gap-6 justify-center">
+            <div className="md:w-[50%] w-full flex justify-start items-center md:gap-6 md:justify-center flex-col ">
+              <ServiceTextTemplate
+                description="Here at HelpStation, we know how important it is to treat your loved one with respect and dignity during their final journey. Our dedicated team will be there for you every step of the way, offering compassionate support and professional hearse services. We want to help you give them the send-off they deserve."
+                headingText="Honoring Your Loved One with Dignity and Respect"
+                heading="Hearse Services"
```
[the remark this answers is not in the document]
Comment (Honeylii) [reply domain/our-services/index.tsx:170]: No , This is just like a description of the service on their service page

**Pull request merged**